website/management/commands/leaderboard.py

```
@@ -25,3 +28,39 @@ def handle(self, *args, **options):
             user_prof.save()
 
         return str("All users updated.")
+
+
+from django.core.exceptions import ObjectDoesNotExist
```

Ruff (0.14.7)

Redefinition of unused `ObjectDoesNotExist` from line 2

Remove definition: `ObjectDoesNotExist`

(F811)